src/messages/mod.rs

```diff
@@ -122,7 +127,7 @@ impl Message {
 
     /// Creates a signed message from single node.
     pub(crate) fn single_src(
```

Comment:
I still see this confusing, there can be only one node with a keypair and that's us. All others are Peers.

Reply:
this passed_in keypair is just used for signing. not being kept and sent_out